docs/source/user-guide/expressions.md

```diff
@@ -22,60 +22,89 @@
 DataFrame methods such as `select` and `filter` accept one or more logical expressions and there are many functions
 available for creating logical expressions. These are documented below.
 
-Expressions can be chained together using a fluent-style API:
+:::{tip}
+Most functions and methods may receive and return an `Expr`, which can be chained together using a fluent-style API:
 
 ```rust
 // create the expression `(a > 6) AND (b < 7)`
 col("a").gt(lit(6)).and(col("b").lt(lit(7)))
 ```
 
+:::
+
 ## Identifiers
 
-| Function | Notes                                        |
-| -------- | -------------------------------------------- |
-| col      | Reference a column in a dataframe `col("a")` |
+| Function   | Explanation                                  |
+| ---------- | -------------------------------------------- |
+| col(ident) | Reference a column in a dataframe `col("a")` |
 
 ## Literal Values
 
-| Function | Notes                                              |
-| -------- | -------------------------------------------------- |
-| lit      | Literal value such as `lit(123)` or `lit("hello")` |
+| Function   | Explanation                                        |
+| ---------- | -------------------------------------------------- |
+| lit(value) | Literal value such as `lit(123)` or `lit("hello")` |
+
+:::{note}
+value
+: A type which implement `Literal`
+:::
+
+## Basic Operators
+
```

Comment:
I think we need to align columns.

operation sounds like a function description, which called Explanataion below and comes as the last column. Would be nice to stick to consistent table format

Reply:
What about this?
- Add `Symbol` column
- Move `Function` column comes before `Method`
- Rename `Operation` to `Explanation` or `Description` maybe?

`| Symbol | Syntax | Function | Method | Explanation |`

Does it work for you?

Reply:
Like in example below the function name comes first and description is last,
In this particular example it looks in reverse order